### DISCUSSION TITLE
Missable flag

### DISCUSSION THREAD

**Discussion by Maximdraco** [RetroAchievements/RAIntegration#899] [Ideas]
I don't know of this is the correct place to give this type of idea, correct me if I'm wrong.

A possible new flag to indicate if a achievement is misseable.

It could works like the "trigger" flag with some signal or indication but for misseables, my idea is than its used with a chain of conditions than indicate than the player is close to the point were you can get the achievement (map, battle, chapter, etc).

I feel than it would requiere to be in a exclusive group apart of the main logic or not be consideres as a condition to trigger the achievement (the same with conditions connected to it).

**Comment by Tsearo** [comment]
#756 discusses this topic a bit with some discussion
